src/app/components/data-detail/data-detail.component.ts

```diff
@@ -36,7 +36,11 @@ export class DataDetailComponent implements OnInit, OnDestroy {
 
     loadData(experimentId: number, dataName: string, version: string | null) {
         this.backend.getExperimentData(experimentId, dataName, version ?? 'latest').pipe(take(1)).subscribe(data => {
-            this.downloadUrl = this.sanitizer.bypassSecurityTrustResourceUrl(this.backend.backendRootUrl + data.download);
+            let url = data.download
+            if (data.download.startsWith("/")) {
```

Comment:
A fancier version would be to check for http(s):// but this should also work fine.

Only make sure, that you never end up withan url like `http://localhost//path` (e.g. by stripping a right slash from the backend URL if there is one. (this can be done in the backend service)